frontends/common/options.cpp

```diff
@@ -145,8 +180,8 @@ FILE* CompilerOptions::preprocess() {
 #else
         std::string cmd("cpp");
 #endif
-        cmd += cstring(" -undef -nostdinc -I") +
-                p4includePath + " " + preprocessor_options + " " + file;
+        cmd += cstring(" -undef -nostdinc -I") + (isv1() ? p4_14includePath : p4includePath)
```

Comment:
I left it unchanged from before, but you're correct, the standard directory should be last in the path (that's the way C compilers do it; gcc has a special -idirafter option to add directories after the standard set)